docs/resources/apig_instance.md

```diff
@@ -102,6 +102,16 @@ The following arguments are supported:
 
   Changing this will create a new resource.
 
+* `vpcep_service_name` - (Optional, String) Specifies the name of the VPC endpoint service.
+  It can contain a maximum of 16 characters, including letters, digits, underscores (_), and hyphens (-).
+  If this parameter is not specified, the system automatically generates a name in the
+  "{region}.apig.{service_id}" format. If this parameter is specified, the system automatically generates a name
+  in the "{region}.{vpcep_service_name}.{service_id}" format.
+  After the gateway is created, you can modify this name on the Gateways > VPC Endpoints page.
```

Comment:
it seems that it's the description for the API response of `vpcep_service_name`

Reply:
yeah, deleted.